.circleci/config.yml

```diff
@@ -637,17 +638,20 @@ workflows:
       - yarn-monorepo
       - js-lint-test:
           name: common-ts-tests
+          coverage_flag: common-ts-tests
```

Comment:
I added this param because I couldn't find a way to access the `name` field above it.